### ISSUE TITLE
[Help needed] Exit status code 255

### ISSUE THREAD

**Issue by sagrawal-code** [silinternational/ecs-deploy#223]
Hello

My deployment (running ecs-deploy via CodeBuild) has started failing since last week with exit status code 255 (no explicit change from me; I do get the latest master version of ecs-deploy on each build though). There is no error message. It deploys fine if I run the same command (using the same image etc.) locally.

I looked at the script and couldn't find anywhere where the exit status code can be 255. If someone can point me to a direction where I can look in order to debug this issue it will be very helpful.

**Comment by sagrawal-code** [comment]
- I was able to gain shell access to Codebuild via AWS Session Manager as described in this article:https://dev.to/glnds/how-to-debug-and-trace-problems-in-aws-codebuild-1cgl
- I discovered that aws ecs commands are failing because of a version conflict between the boto libraries (boto3 and botocore) installed by awscli and the ones which were being installed by the application's test runner for running unit tests.
- I upgraded the boto3 package in the application's requirements file.
- This removed the version conflict and the aws ecs command and hence, ecs-deploy was able to run successfully again.

It might be good to have a "--debug" flag to ecs-deploy which returns the debug output of a failed awscli command.

Thank you!
